package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "terra-aggregate-translations",
+  "version": "0.0.0",
+  "description": "The terra-aggregate-translations pre-build tool assists with creating the translation, intl loader and translation loader files that are configured for the specified locales. This tool is offered as a CLI script and as a setup function.",
+  "main": "lib/index.js",
+  "engines": {
+    "node": ">=8.9.2"
```

Comment:
Lets lock this down to match current engine versions in other terra repos: 
`>=8.9.2 <12`

Reply:
Updated: https://github.com/cerner/terra-aggregate-translations/pull/1/commits/e30c662bfdecaf4554f9cd135bc9552e4d933978